pkg/client/scanner/scanner.go

```diff
@@ -31,44 +35,52 @@ type Config struct {
 }
 
 // Available list all available scanner tools that can be used.
-var Available = [...]Tool{
-	Licensee,
-	Scancode,
-	Composer,
-	FileHasher,
-}
+var Available []Tool
 
 // Default is the default scanner tools that shall be used if no particular tool is selected.
-var Default Tool = Licensee
+var Default Tool
+
+// init loads available scanner tools from config file. It also assigns default tool
+func init() {
+
+	jsonFile, err := os.Open("pkg/client/scanner/scanner_tools.json")
```

Comment:
Please remove the second `os.Open()` inside the if block.
Use the path `./scanner_tools.json` as the only one.

Reply:
go test uses the directory containing the source file instead of root directory, my next commit will fix this